pkg/kube/kube_handler.go

```diff
@@ -47,6 +47,7 @@ import (
 type ClusterConfig struct {
 	Name                     string `json:"name"`
 	APIServiceURL            string `json:"apiServiceURL"`
+	APIServicePath           string `json:"apiServicePath"`
```

Comment:
@absoludity the idea here is I have created a separate parameter for the k8s API service "path" (e.g. the `/k8s` in my setup).

This way I can simply concatenate `APIServiceURL` with `APIServicePath` here in the kubeops backend code, and conditionally write the NGINX rewrite rules (see above in this diff).